parser/shared/src/main/scala/hmda/parser/fi/lar/LarCsvParser.scala

```diff
@@ -0,0 +1,98 @@
+package hmda.parser.fi.lar
+
+import hmda.model.fi.lar._
+
+object LarCsvParser {
+  def apply(s: String): LoanApplicationRegister = {
+    val values = s.split('|').map(_.trim)
```

Comment:
Agreed on the parsing library. There are no legit reasons for `|` characters that I know of, certainly not for the LAR part. If there are, we need another separator.